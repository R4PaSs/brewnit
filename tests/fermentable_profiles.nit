import dummy_db

with ctx = new DBContext do
	var res = ctx.fermentable_profile_worker.fetch_multiple(ctx, "* FROM fermentable_profiles")
	for i in res do print i
end
