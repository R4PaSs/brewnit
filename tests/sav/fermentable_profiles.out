Fermentable: Grain: Two-Row, potential: 1.038 SG, colour: 2.0 SRM, quantity: 4500.0 g
Fermentable: Grain: Crystal 40, potential: 1.037 SG, colour: 40.0 SRM, quantity: 500.0 g
Fermentable: Adjunct: Pumpkin, potential: 1.0 SG, colour: 0.0 SRM, quantity: 2267.962 g
Fermentable: Adjunct: Cinnamon, potential: 1.0 SG, colour: 0.0 SRM, quantity: 1.5 g
Fermentable: Adjunct: Cloves, potential: 1.0 SG, colour: 0.0 SRM, quantity: 0.5 g
Fermentable: Adjunct: Ginger, potential: 1.0 SG, colour: 0.0 SRM, quantity: 0.5 g
Fermentable: Adjunct: Nutmeg, potential: 1.0 SG, colour: 0.0 SRM, quantity: 0.5 g
Fermentable: Grain: Two-Row, potential: 1.038 SG, colour: 2.0 SRM, quantity: 2500.0 g
Fermentable: Grain: Flaked Oats, potential: 1.037 SG, colour: 1.0 SRM, quantity: 250.0 g
Fermentable: Grain: Flaked Wheat, potential: 1.035 SG, colour: 1.6 SRM, quantity: 1250.0 g
Fermentable: Grain: Wheat Malt, potential: 1.036 SG, colour: 2.0 SRM, quantity: 1500.0 g
Fermentable: Adjunct: Rice Hulls, potential: 1.0 SG, colour: 0.0 SRM, quantity: 400.0 g
Fermentable: Adjunct: Hibiscus/Berries Tisane, potential: 1.0 SG, colour: 0.0 SRM, quantity: 200.0 g
Fermentable: Grain: Two Row, potential: 1.037 SG, colour: 1.981 SRM, quantity: 5600.0 g
Fermentable: Extract: Light liquid extract, potential: 1.034 SG, colour: 1.981 SRM, quantity: 1500.0 g
Fermentable: Grain: Crystal 120, potential: 1.034 SG, colour: 120.0 SRM, quantity: 350.0 g
Fermentable: Grain: Roasted Barley, potential: 1.029 SG, colour: 400.0 SRM, quantity: 250.0 g
Fermentable: Grain: Chocolate Malt, potential: 1.03 SG, colour: 300.0 SRM, quantity: 900.0 g
Fermentable: Adjunct: Star Anise, potential: 1.0 SG, colour: 0.0 SRM, quantity: 30.0 g
